internal/template/template.go

```diff
@@ -181,6 +181,32 @@ func (mock *{{$mock.MockName}}
 	mock.lock{{.Name}}.RUnlock()
 	return calls
 }
+{{- if $.WithResets}}
+	// {{.Name}}ResetCalls Reset all the calls that were made to {{.Name}}.
```

Comment:
what if this is just `Reset()`?

Reply:
I think `ResetCalls()` is less likely to cause a collition with an actual method of the mocked interface.